Codeinwp/neve
# Advanced search performance improvement

### Summary
- Changed FormTokenField control to make requests for post suggestions while the user is typing.
- Added Rest parameter to be able to search by "post title starts with"
- Removed the multi-select control that was previously added

### Will affect the visual aspect of the product
NO


### Test instructions
- Please test this version with Neve 3.4.1. The Advanced search components should not exist at all unless you update both of them
- Using Neve 3.4.1 and Neve Pro 3.4.1 configure advanced search components. Then update both versions and make sure they still work and the data is still there.


Closes Codeinwp/neve-pro-addon#2171.
